src/flux/course.coffee

```diff
@@ -46,7 +47,7 @@ CourseConfig =
     @emit('practice.loaded', obj.id)
 
   _reset: ->
-    CrudConfig.reset.call(@)
+    CrudConfig.reset.call(this)
```

Comment:
i see.  also good job on not accidentally linking, i totally did that on the react-calendar stuff lol.